docs/advanced-features/rust-compiler.md

```diff
@@ -0,0 +1,139 @@
+---
+description: Learn about the Rust Compiler, built with SWC, which transforms and minifies your Next.js application.
+---
+
+# Rust Compiler
```

Comment:
I think this is misleading since users cannot use Rust. 

For example, Next.js ships with a TypeScript compiler so users can author TS code. But Next.js doesn't ship with a Rust compiler.

Maybe we should rename this whole document to `Speedy Web Compiler (SWC)`, especially if we plan to use it for TS